changelog/10643.misc.md

```diff
@@ -0,0 +1 @@
+Add a schema for single events.
```

Comment:
Could you please add the motivation for this refactoring please, either in the changelog or the PR description?

Reply:
good point!!